cmd/sqlbee/mutate.go

```diff
@@ -123,6 +124,14 @@ func mutatePodSpec(volumes []corev1.Volume, proxyContainer *corev1.Container, po
 			podSpec.Containers = append(podSpec.Containers[:i], podSpec.Containers[i+1:]...)
 			break
 		}
+		container.Resources.Limits = corev1.ResourceList{
+			corev1.ResourceMemory: resource.MustParse("100Mi"),
+			corev1.ResourceCPU:    resource.MustParse("30m"),
+		}
+		container.Resources.Requests = corev1.ResourceList{
```

Comment:
This looks ok, but I would like to ask you to create an issue that we should probably make these values configurable.

Reply:
Values  are now configurable with flags